Makefile

```diff
@@ -154,6 +154,14 @@ endif
 
 	@echo plugin built at: dist/$(BUNDLE_NAME)
 
+info: ## Display build information
```

Comment:
This was only added for debugging purposes. Will remove this after the plugin becomes stable. 
Created the issue to track this down: https://github.com/mattermost/mattermost-plugin-boards/issues/3